<?xml version="1.0"?>
<launch>

    <node name="gate_detection_node"    pkg="gate_detection"     type="gate_detection_node.py"    output="screen"/>
    <node name="gate_detection_cfg"    pkg="gate_detection"     type="dynam_reconfigure_server.py"    output="screen"/>

    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="detection" args="0 0 0 0 0 0 1 map detection" /> -->

</launch>